programs/programs/trua/calculator.py

```diff
@@ -0,0 +1,37 @@
+from programs.programs.calc import Eligibility, ProgramCalculator
+import programs.programs.messages as messages
+
+
+class Trua(ProgramCalculator):
+    income_limits = {
+        1: 66_300,
+        2: 75_750,
+        3: 85_200,
+        4: 94_560,
+        5: 102_250,
+        6: 109_800,
+        7: 117_400,
+        8: 124_950,
+    }
+
```

Comment:
We have a partner that sends people to MFB results with a partial screen. We use the `dependencies` to determine if we can calculate the results with the missing fields.
```suggestion
    dependencies = ['income_amount', 'income_frequency', 'household_size', 'zipcode']
```